pkg/ccl/backupccl/backup_processor_planning.go

```diff
@@ -194,6 +196,10 @@ func distBackup(
 			// Send the progress up a level to be written to the manifest.
 			progCh <- meta.BulkProcessorProgress
 		}
+
+		if meta.AggregatorEvents != nil {
+			tracingAggCh <- meta.AggregatorEvents
+		}
```

Comment:
I wonder if we want a layer of safety here against tracing ever blocking.  Namely, I'm wondering if we want some reasonable sized buffer. If the buffer is full, perhaps then we send with some timeout. If we hit the timeout, we drop the event on the floor with a warning.  If you agree, perhaps we can do that in a follow up PR to not block this chain of PRs you have built up.

Reply:
I think this is a good idea, I will file an issue and follow up with this.